README.md

```diff
@@ -112,4 +105,4 @@ go install; go test -v -p 1 $(go list ./... | grep -v  '/vendor/')
 
 ## Further reading
 * [Step-by-step send payment guide with docker](https://github.com/lightningnetwork/lnd/tree/master/docker)
-* [Contribution guide](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
\ No newline at end of file
+* [Contribution guide](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
```

Comment:
Hmm, yeah not sure what happened here. Visually, there's no diff at all created by this line.